docs/source/scancode-toolkit/how_to/how_to_run_a_scan.rst

```diff
@@ -6,46 +6,56 @@ How to Run a Scan
 Quickstart
 ----------
 
-ScanCode results are provided as:
-
 
-#. JSON file (default)
-#. html (static html)
+ScanCode results are provided as:
 
-The basic usage is:
+1. JSON file (default)
+2. html (static html)
+3. csv
 
-::
+The basic usage is::
 
-   ./scancode [OPTIONS] <input> <output_file>
+    ./scancode [OPTIONS] <OUTPUT FORMAT(S)> <input>
 
+  OUTPUT FORMAT(S)::
+    --json FILE             Write scan output as compact JSON to FILE.
+    --json-pp FILE          Write scan output as pretty-printed JSON to FILE.
+    --json-lines FILE       Write scan output as JSON Lines to FILE.
+    --csv FILE              Write scan output as CSV to FILE.
+    --html FILE             Write scan output as HTML to FILE.
+    --custom-output FILE    Write scan output to FILE formatted with the custom
+                            Jinja template file.
+    --custom-template FILE  Use this Jinja template FILE as a custom template.
+    --spdx-rdf FILE         Write scan output as SPDX RDF to FILE.
+    --spdx-tv FILE          Write scan output as SPDX Tag/Value to FILE.
+    --html-app FILE         (DEPRECATED: use the ScanCode Workbench app instead
+                            ) Write scan output as a mini HTML application to
+                            FILE.
 
-**Note: On Windows use scancode instead of ./scancode**
+The ``<input>`` file or directory is what will be scanned for origin clues. The results will be saved to the ``FILE``.
 
-The ``<input>`` file or directory is what will be scanned for origin clues. The results will be saved to the ``<output_file>``.
-The output file format is set by using the ``-f`` or ``--format`` option. The default output format is JSON.
 
 The following example scans will show you how to run a scan with each of the result formats. For the scans, we will use the ``samples`` directory provided with the ScanCode Toolkit.
 
-JSON file output
-----------------
-
-Scan the ``samples`` directory and save the scan to a JSON file:
+**Note: On Windows use scancode instead of ./scancode**
 
-::
+**JSON file output**
 
-   ./scancode --format json samples samples.json
+Scan the ``samples`` directory and save the scan to a sample JSON file::
 
+    ./scancode --json samples.json samples
```

Comment:
This will give an error `Error: no such option: --json` because scancode does not have a `--json` flag. You can again check the `./scancode --help` for all the available options. Also, this doesn't scan a folder named ``samples`` but scans a samples.json file instead, the difference is clearly obvious. The correct format was in the previously existing file.